docs/_manage/integration-performance/user-feedback.md

```diff
@@ -99,4 +99,28 @@ Any reported issue with no updates from you, Zapier Support, or end users for a
 
 - Usage details for each trigger and action will show which functionalities are the most popular and being effectively utilized. 
 
-- Share access to the insights by [inviting collaborators](https://platform.zapier.com/manage/invite-team-member) to the integration without giving them permissions to make changes to it.
\ No newline at end of file
+- Share access to the insights by [inviting collaborators](https://platform.zapier.com/manage/invite-team-member) to the integration without giving them permissions to make changes to it.
+
+## Utilizing issue statuses
+
+Maintaining accurate issue statuses during regular triage and review of your integration's issues is critical for keeping your integration healthy. Not only do statuses help your team track progress and build roadmaps, but they also signal your engagement and prompt reviews for fixed issues to the Zapier team.
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="500" src="https://on.driveway.app/guides/zEVVdGE/embed" allow="clipboard-write" allowfullscreen></iframe>
+
```

Comment:
I might add some helptext under the iframe like (Enter fullscreen for a better experience). 

I know it was weird looking just in the normal size and it took me a minute to figure out there was a fullscreen option. 

Also feel free to disregard if this is a me issue.